tests/unit/FilingHistoryList.spec.ts

```diff
@@ -388,13 +388,13 @@ describe('Filing History List - misc functionality', () => {
     expect(vm.disableCorrection({ ...item, isFutureEffective: true })).toBe(true)
 
     // only fifth condition:
-    expect(vm.disableCorrection({ ...item, status: 'CORRECTED' })).toBe(true)
+    expect(vm.disableCorrection({ ...item, status: 'CORRECTED' })).toBe(false)
```

Comment:
Why is this now expected to be false?

Reply:
I misunderstood it. I've reverted the two lines.